virtool/labels/api.py

```diff
@@ -21,12 +22,11 @@ async def find(req):
     Get a list of all label documents in the database.
 
     """
-    documents = list()
-    async with AsyncSession(req.app["pg"]) as session:
-        result = await session.execute(select(Label))
-        labels = result.scalars().all()
-        for label in labels:
-            documents.append(label.to_dict())
+    find = req.query.get("find")
+
+    labels = await get_rows(req.app["pg"], find, Label)
+
+    documents = [label.to_dict() for label in labels]
 
     documents = await asyncio.gather(*[virtool.labels.db.attach_sample_count(req.app["db"], d) for d in documents])
```

Comment:
Suggestion: combine so `documents` is iterated only once.

Do `from virtool.labels.db import attach_sample_count` to make line more readable.